src/gql/generated/types.ts

```diff
@@ -13,339 +13,159 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
+  Time: Date;
```

Comment:
You can remove this file since your PR didn't require adjusting the gql schema or any queries